controller/work_item_link_type.go

```diff
@@ -18,8 +18,9 @@ import (
 // WorkItemLinkTypeController implements the work-item-link-type resource.
 type WorkItemLinkTypeController struct {
 	*goa.Controller
-	db     application.DB
-	config WorkItemLinkTypeControllerConfiguration
+	db           application.DB
+	config       WorkItemLinkTypeControllerConfiguration
+	configCombis WorkItemLinkTypeCombinationControllerConfiguration
```

Comment:
The `WorkItemLinkTypeCombinationControllerConfiguration` is used in two controllers: `WorkItemLinkTypeCombinationController` and `WorkItemLinkTypeController`. That's why I would like to keep them separate.